techeria_controller/templates/index.html

```diff
@@ -451,10 +450,10 @@ <h5>
             </div>
             <div class="detail_box">
               <h5>
-                24x7 Customer support

```

Comment:
We also have to make sure the contact form is available to access from the home page!